__tests__/crontab.test.ts

```diff
@@ -237,3 +237,28 @@ describe("gives error on syntax error", () => {
     );
   });
 });
+
+describe("parses JS cron items correctly", () => {
+  test("defaults backfillPeriod to 0", () => {
+    expect(
+      parseCronItem({
+        task: "task",
+        match: "* * * * *",
+        options: {},
+      }).options.backfillPeriod,
+    ).toEqual(0);
+  });
+
+  test("validates match strings", () => {
+    const matchString = "foo";
+    expect(() =>
+      parseCronItem({
+        task: "task",
+        match: matchString,
+        options: {},
+      }),
+    ).toThrowErrorMatchingInlineSnapshot(
```

Comment:
An inline snapshot can but updated by `jest -u`; but yours includes a variable? 
```suggestion
    ).toThrow(
```
https://jestjs.io/docs/expect#tothrowerror

Reply:
Shoot. Careless copilot autocomplete. Thanks for addressing